docs/becoming-a-maintainer/getting-practical.md

```diff
@@ -140,7 +140,10 @@ Thank you for contributing to [Project Name]!
 
 ### Setting Up a Code of Conduct
 
-It's essential to have a code of conduct to ensure that everyone feels welcome and safe in your project. Here's how to set up a code of conduct:
+It's essential to have a code of conduct to ensure that everyone feels welcome and safe in your project. There are two ways to add this file to your project:
+
+1. Use a premade template: GitHub provides a code of conduct template. To learn more, check out their tutorial, ["Adding a code of conduct using a template"](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-using-a-template).
+2. Manual: If you want to create this file on your own, follow these steps:
```

Comment:
- As GitHub docs doesn't mention "premade", let's follow their docs.
- The grammar for using a template is incorrect here. There are a couple of CoC templates on GitHub.
- I think it's "Manually" instead of "Manual" here.
- Because the 1st method begins with a verb, we can also start with a verb for the 2nd one, as this seems possible.

```suggestion
1. **Use a template**. GitHub provides common code of conduct templates to help you set one up quickly. You can read [the instructions on their docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-using-a-template) to add one to your project.
2. **Add manually**. If you want to copy or adapt from other codes of conduct, you need to add your code of conduct manually by following these steps:
```